AlphaWallet.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -5,35 +5,35 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/firebase/abseil-cpp-SwiftPM.git",
       "state" : {
-        "revision" : "fffc3c2729be5747390ad02d5100291a0d9ad26a",
-        "version" : "0.20200225.4"
+        "revision" : "d302de612e3d57c6f4afaf087da18fba8eac72a7",
```

Comment:
PR looks ok, but why did the dependencies change?

Reply:
added by mistake, removed it